<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account_create_container</class>
 <widget class="QWidget" name="account_create_container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 36, 35);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QWidget" name="username_container" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="user_label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>User Name / Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="user_input">
        <property name="styleSheet">
         <string notr="true">border: 'none';
background-color: rgb(51, 53, 51);
padding: 5;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="password_container" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="password_label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_input">
        <property name="styleSheet">
         <string notr="true">border: 'none';
background-color: rgb(51, 53, 51);
padding: 5;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="platform_container" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="platform_label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Platform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="platform_input">
        <property name="styleSheet">
         <string notr="true">border: 'none';
background-color: rgb(51, 53, 51);
padding: 5;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="website_container" native="true">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="website_label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Website</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="website_input">
        <property name="styleSheet">
         <string notr="true">border: 'none';
background-color: rgb(51, 53, 51);
padding: 5;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="create_container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="Create_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(245, 203, 92);
color: rgb(36, 36, 35);
border: 'none';
padding: 5;
padding-right: 15;
padding-left: 15;</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
